src/table/table.scss

```diff
@@ -4,383 +4,373 @@
+      @include themed {
+        border-left-color: t(iui-color-background-4);
+        border-right-color: t(iui-color-background-4);
+        border-bottom-color: transparent;
       }
 
-      &.iui-active-sort {
-        @include themed {
-          background-color: t(iui-color-background-4);
-        }
+      .iui-row-expander > .iui-icon {
+        transform: rotate(90deg);
+      }
 
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          visibility: visible;
+      + .iui-expanded-content {
+        @include themed {
+          border-left-color: t(iui-color-background-4);
+          border-right-color: t(iui-color-background-4);
         }
       }
     }
 
-    &.iui-tables-row-active {
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected,
+    &.iui-selected:first-child {
+      border-bottom-color: transparent;
+    }
+
+    &.iui-selected {
       @include themed {
-        border: solid 1px t(iui-color-foreground-primary);
+        border-color: t(iui-color-foreground-primary);
         background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
       }
-    }
 
-    &.iui-tables-row-new {
-      font-weight: $iui-font-weight-semibold;
+      + .iui-selected {
+        border-bottom-color: transparent;
+        @include themed {
+          border-top-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
+        }
+      }
 
-      > .iui-tables-filename {
-        position: relative;
+      &:last-child {
+        @include themed {
+          border-bottom-color: t(iui-color-foreground-primary);
+        }
       }
 
-      > .iui-tables-filename::before {
+      + :not(.iui-selected),
+      + .iui-expanded-content + :not(.iui-selected) {
         @include themed {
-          color: t(iui-color-foreground-positive);
+          border-top-color: t(iui-color-foreground-primary);
         }
+      }
+
+      + .iui-expanded-content {
+        @include themed {
+          border-color: transparent t(iui-color-foreground-primary);
+        }
+      }
+    }
+    // #endregion Selection
+
+    // #region Statuses & states
+    &.iui-new {
+      font-weight: $iui-font-weight-semibold;
+      
+      > .iui-main-column::before {
         content: '•';
         position: absolute;
         left: 0;
         font-size: $iui-font-size-title;
+        @include themed {
+          color: t(iui-color-foreground-positive);
+        }
       }
     }
 
-    &.iui-tables-row-disabled {
+    &.iui-disabled,
+    &.iui-disabled + .iui-expanded-content {
+      font-style: italic;
+      cursor: not-allowed;
       @include themed {
         color: t(iui-text-color-muted);
       }
-      font-style: italic;
-      cursor: not-allowed;
 
-      &:hover {
-        @include themed {
-          background-color: t(iui-color-background-1);
-        }
-
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
-        }
+      &:hover > .iui-slot > .iui-more-options {
```

Comment:
Found a strange bug. When clicked on expansion icon in disabled row then moving mouse to an other row, menu icon appears.
![disabled-row-menu](https://user-images.githubusercontent.com/36186912/121473494-709fd880-c9cb-11eb-8888-76e5ce853c16.gif)

Reply:
😮  Must be something with focus

Reply:
This has to do with `focus-within` on line 153.  This seems to work:
```
 &:not(.iui-disabled):focus-within > .iui-slot > .iui-more-options {
    visibility: visible;
}
```

Reply:
fixed